src/middlewared/middlewared/plugins/catalogs_linux/apps.py

```diff
@@ -8,7 +8,7 @@ class AppService(Service):
     class Config:
         cli_namespace = 'app'
 
-    @filterable
+    @filterable(roles=['READONLY', 'CATALOG_READ'])
```

Comment:
You don't need to add `READONLY` and `CATALOG_READ` here if you add `CATALOG_READ` as a member of `READONLY`.